plugins/plugin-clangd/che-plugin-clangd-lang-server/src/main/java/org/eclipse/plugin/clangd/inject/ClangModule.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.plugin.clangd.inject;
+
+import static java.util.Arrays.asList;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.multibindings.Multibinder;
+import org.eclipse.che.api.languageserver.launcher.LanguageServerLauncher;
+import org.eclipse.che.api.languageserver.shared.model.LanguageDescription;
+import org.eclipse.che.api.project.server.type.ProjectTypeDef;
+import org.eclipse.che.inject.DynaModule;
+import org.eclipse.che.plugin.cpp.projecttype.CProjectType;
+import org.eclipse.che.plugin.cpp.projecttype.CppProjectType;
+import org.eclipse.plugin.clangd.languageserver.ClangDLanguageServerLauncher;
+
+/** @author Alexander Andrienko */
+@DynaModule
+public class ClangModule extends AbstractModule {
+  public static final String LANGUAGE_ID = "clangd";
+  private static final String[] EXTENSIONS =
+      new String[] {
+        "c", "h", "cpp", "hpp", "cc", "hh", "hxx", "cxx", "C", "H", "CPP", "HPP", "CC", "HH", "CXX",
+        "HXX"
+      };
+  private static final String MIME_TYPE = "text/x-cpp";
+
+  // @Override
```

Comment:
I just would like to remind that we're trying to avoid having comments of such kind when we do merge to master, do you think this section could be removed?

Reply:
good point, i will go through it and remove those